packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -284,7 +284,7 @@ class WritingFlow extends Component {
 		if ( ! isVertical ) {
 			this.verticalRect = null;
 		} else if ( ! this.verticalRect ) {
-			this.verticalRect = computeCaretRect( target );
```

Comment:
Why did this change? I guess because `computeCaretRect` doesn't really do much with the `target` argument?

Reply:
Yeah, it's not very useful plus it allows us to use `computeCaretRect` in case with don't know the container. It doesn't really need to know the container to calculate the rect.